tests/core/test_core_rpc_handlers.py

```diff
@@ -45,6 +53,70 @@ async def test_rpc_nodeInfo(make_request, enr):
     assert node_info["enr"] == repr(enr)
 
 
+@pytest.mark.trio
+async def test_rpc_updateNodeInfo(make_request, enr_manager):
+    # add a kv pair
+    first_response = await make_request("discv5_updateNodeInfo", [("foo", "bar")])
+    assert first_response is None
+    assert enr_manager.enr._kv_pairs[b"foo"] == b"bar"
+    assert enr_manager.enr.sequence_number == 2
+
+    # update a kv pair
+    second_response = await make_request("discv5_updateNodeInfo", [("foo", "xyz")])
+    assert second_response is None
```

Comment:
What do you think about returning the updated ENR from `discv5_updateNodeInfo`

Reply:
Yeah that seems reasonable - I was just following the API from `ENRManager.update` here.

Reply:
Back when I wrote `ENRManager.update` I certainly considered returning the updated ENR record, but chose against it due to ... the position of Saturn and Mercury being in retrograde?

Reply:
I've been there before - it also happens to be the cause of every bug I've written - darn retrograde.